docs/src/content/docs/id/reference/configuration.mdx

```diff
@@ -346,6 +379,94 @@ starlight({
 });
 ```
 
+### `expressiveCode`
+
+**type:** `StarlightExpressiveCodeOptions | boolean`  
+**default:** `true`
+
+Starlight menggunakan [Kode Ekspresif](https://github.com/expressive-code/expressive-code/tree/main/packages/astro-expressive-code) untuk merender blok kode dan menambahkan dukungan untuk menyorot bagian contoh kode, menambahkan nama file ke blok kode, dan banyak lagi.
```

Comment:
Regarding "Kode Ekspresif", Expressive Code is the name of the library and should not be translated (I think this should be fixed 5 times below if I'm correct).

Reply:
Okay, will update all the "Expressive Code" translations immediately.